app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class TasksController < ApplicationController
+
+  def index
+    if params[:search] 
+      @tasks = Task.search(params[:search])
+    else
+      @tasks = Task.order(completed_at: :desc, created_at: :asc)
+    end
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def new
+    @task = Task.new
+    @people = Person.by_name
+  end
+
+  def create
+    @task = Task.new(tasks_create_params[:task])
+    if @task.save
+      redirect_to tasks_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @task = Task.where(id: params[:id]).first
+    @people = Person.by_name
```

Comment:
I believe you could use Task.find(params[:id]) here since you shouldn't get more than one Task with the same ID. It would be a little more consistent with how you're doing the show pages.